packages/stk/stk_balance/stk_balance/CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ TRIBITS_ADD_EXECUTABLE(
     NOEXEPREFIX INSTALLABLE
     )
 
-TRIBITS_ADD_EXECUTABLE_AND_TEST(
+TRIBITS_ADD_EXECUTABLE(
```

Comment:
This removes this as a test, right?

Reply:
Yes. It wasn't supposed to be a test. stk_balance has unit-tests under stk/stk_unit_tests/stk_balance. This is the user-facing executable.